src/transformers/modeling_utils.py

```diff
@@ -4292,7 +4297,7 @@ def from_pretrained(
                 dispatch_model(model, **device_map_kwargs)
 
         if hf_quantizer is not None:
-            hf_quantizer.postprocess_model(model)
+            hf_quantizer.postprocess_model(model, config=config)
```

Comment:
need config for decompression -> find the filepath of where the model.safetensors